v3/src/components/graph/models/data-configuration-model.ts

```diff
@@ -119,41 +124,47 @@ export const DataConfigurationModel = types
       return selection.filter(caseId => self.filteredCases?.hasCaseId(caseId))
     }
   }))
-  .views(self => ({
-    attributeValuesForPlace(place:GraphAttrPlace):string[] {
-      const attrID = self.attributeID(place),
-        dataset = self.dataset
-      return attrID ? self.cases.map(anID => String(dataset?.getValue(anID, attrID)))
-        .filter(aValue => aValue !== '') : []
-    },
-    categorySetForPlace(place:GraphAttrPlace):Set<string> {
-      const result:Set<string> = new Set(this.attributeValuesForPlace(place))
-      result.delete('')
-      if(result.size === 0) {
-        result.add('__main__')
+  .views(self => (
+    {
+      attributeValuesForPlace(place: GraphAttrPlace): string[] {
+        const attrID = self.attributeID(place),
+          dataset = self.dataset
+        return attrID ? self.cases.map(anID => String(dataset?.getValue(anID, attrID)))
+          .filter(aValue => aValue !== '') : []
+      },
+      numAttributeValuesForPlace(place: GraphAttrPlace):number[] {
```

Comment:
```suggestion
      numericValuesForPlace(place: GraphAttrPlace):number[] {
```
At the very least, I would spell out `numeric` here because otherwise it could be interpreted as retrieving the number of values. If brevity is the goal, then perhaps `Attribute` can be removed, resulting in `valuesForPlace()` and `numericValuesForPlace()`.